sdk/subscription/Azure.ResourceManager.Subscription/src/autorest.md

```diff
@@ -16,9 +16,34 @@ skip-csproj: true
 modelerfour:
   flatten-payloads: false
 
+mgmt-debug: 
+  show-serialized-names: true
```

Comment:
Why do we need to show the serialized name in comment?

Reply:
fixed